app/controllers/api/base_controller/authentication.rb

```diff
@@ -147,7 +147,7 @@ def valid_ui_session?
 
       def authenticate_with_jwt
         timeout = ::Settings.api.authentication_timeout.to_i_with_method
```

Comment:
the setting of the timeout now is not needed, right?

Reply:
Good catch @chessbyte,  but backwards. I had avoided using the timeout to aid in getting things working. The timeout should be used, adding use of the timeout back.